Find_The_Largest_Island.py

```diff
@@ -0,0 +1,209 @@
+from typing import List, Tuple, Generator
+from collections import deque
+
+# Constants
+LAND = 1
+WATER = 0
+
+DIRECTIONS = [(-1, 0), (1, 0), (0, -1), (0, 1)]
+
+
+class Coordinates:
+    """Single coordinates on a map."""
+    def __init__(self, row: int, col: int) -> None:
+        self.row = row
+        self.col = col
```

Comment:
Dla czytelności polecam ustawiać atrybyty których będziesz używać na poziomie klasy (a potem tylko przypisywać do nich w init), możesz też zobaczyć co robi `__slots__` bo czasem o to ktoś może spytać, oraz https://docs.python.org/3/library/dataclasses.html które często są używane do budowania takich obiektów

```suggestion
class Coordinates:
    """Single coordinates on a map."""
    row: int
    col: int
    
    def __init__(self, row: int, col: int) -> None:
        self.row = row
        self.col = col
```